Poker.BE/Poker.BE.Domain/Utility/Exceptions/NotEnoughPlayersException.cs

```diff
@@ -4,7 +4,7 @@
 namespace Poker.BE.Domain.Utility.Exceptions
 {
     [Serializable]
-    internal class NotEnoughPlayersException : Exception
+    public class NotEnoughPlayersException : Exception
```

Comment:
make it drive from `PokerException` class. if you don't have it now, you will have it with the next merge with `idan_ucc03_1` branch, or `master` - depends if i manage to finish my work before your next pull.